trin-storage/src/versioned/id_indexed_v1/store.rs

```diff
@@ -80,8 +80,11 @@ impl VersionedContentStore for IdIndexedV1Store {
 
         let mut store = Self {
             config,
-            to_insert_until_pruning: 0,
             radius: Distance::MAX,
+            usage_stats: UsageStats {
```

Comment:
*nitpick* seems like this could be replaced with a `UsageStats::new()` implementation